ads/readmo.js

```diff
@@ -21,26 +21,21 @@ import {loadScript, validateData} from '../3p/3p';
  * @param {!Object} data
  */
 export function readmo(global, data) {
-  validateData(
-    data,
-    ['section', 'url'],
-    ['module', 'sponsoredByLabel', 'infinite', 'title']
```

Comment:
Did the optional attrs not work with JSON?

Reply:
Yeah 😕It was throwing validations errors for any property defined in the JSON.

Reply:
thanks for the info! will look into it.